bazel/test/python_test_repo/.bazelrc

```diff
@@ -0,0 +1,3 @@
+build:python3 --python_path=python3
```

Comment:
Just wanted to add that there's [a PR](https://github.com/grpc/grpc/pull/20124/files) up right now that might result in merge conflicts with this.

Reply:
Can you please clarify your comment? This is `.bazelrc` file for the test "sub-workspace". 

This is needed, because `py_proot_library` rules break on our workstations with bazel 0.27.0+.Starting from this version Bazel started resolving python differently and instead of resolving to Python 2.7 by default it resolves to Python 3.6. This breaks python builds, because of the `enum34` dependency (imported here https://github.com/grpc/grpc/blob/master/bazel/grpc_python_deps.bzl#L25 (which is not supported after Python3.4 or something and conflicts with the built in enum in python)).

Reply:
Right. We dealt with this problem in the PR I linked above. And part of the solution is to explicitly removing the usage of the `python_path` flag. All that should be necessary now is to remove the subworkspace's `.bazelrc` and to explicitly add a `python_version` attribute to the tests.

The diff is a bit messed up in the unreversion PR I linked, but in [the original PR](https://github.com/gnossen/grpc/blob/e70788364bf38947e9d3afb820800b14e44829b9/bazel/python_rules.bzl#L183), we switched to using a custom `py2and3_test` for every test.

Reply:
Switched to py2and3test and removed `.bazelrc` file from the build